Source.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+#include <cstdlib>
+#include "Character.h"
+
+
+using namespace std;
+
+void auto_combat(Character& player1, Character& player2){
+	int turn=1;
+	bool end=false;
+	while (!end){
+		player1.stat_out();player2.stat_out();
+		if (turn==1){
+				cout<<player1.getName()<<" -> "<<player2.getName()<<endl;
+				player1.attack(player2);
+				turn=2;
+				if (player2.isDead()){
+					end=true;
+					player1.stat_out();player2.stat_out();	
+					cout<< player2.getName()<<" died. "<< player1.getName()<<" wins.\n";
+				}
+		}
+			else if (turn==2){
+				cout<<player2.getName()<<" -> "<<player1.getName()<<endl;
+				player2.attack(player1);
+				turn=1;
+				if (player1.isDead()){
+					end=true;
+					player1.stat_out();player2.stat_out();
+					cout<< player1.getName()<<" died. "<< player2.getName()<<" wins.\n";
+				}
+			}
+			else{
+				cerr<<"Turn can only be 1 or 2!";
+			}
+	}	
+}
```

Comment:
itt az indentalast tegyetek rendbe, kisse atlathatatlan igy.

Reply:
Hozzáadtam a take_turn függvényt hogy jobban átlátható legyen, ne legyen annyi kódismétlés